src/api.js

```diff
@@ -0,0 +1,35 @@
+const API_URL = 'https://bootcamp-api.codeit.kr/api';
+const userId = 1;
+
+export function fetchData(url) {
+  return fetch(`${API_URL}/${url}`).then(res => {
+    if (!res.ok) {
+      throw new Error('데이터를 불러오는데 실패했습니다.');
+    }
+    return res.json();
+  });
+}
+
+export function getSampleUser() {
+  return fetchData('sample/user');
+}
+
+export function getSampleFolder() {
+  return fetchData('sample/folder');
+}
+
+export function getUsers() {
+  return fetchData(`users/${userId}`);
+}
+
+export function getFolderTabs() {
+  return fetchData(`users/${userId}/folders`);
+}

```

Comment:
26L으로 가져온 folder 목록 데이터를 어떻게 사용할지(탭으로 표현할지)는 반환받은 쪽에서 결정되고, 언제든 달라질 수 있으니, `Tabs`라는 '어떻게 사용될지'의 맥락은 빠지는 편이 좋겠습니다

```suggestion
export function getFolders() {
```